src/events/components/DetailView/Contact.tsx

```diff
@@ -7,14 +7,16 @@ import style from './detail.less';
 const Contact = ({ event_type, organizer_name, company_event }: INewEvent) => {
   const color = getEventColor(event_type);
 
+  const organizer_email = `${organizer_name.toLowerCase()}@online.ntnu.no`;
```

Comment:
Make it camelCase, and I'll merge 😄

Reply:
But that doesn't make sense when all the other variables are in snake_case?

Reply:
While I agree that it is a bit strange, it is a result of our current technologies.

JavaScript and TypeScript use camelCase, and we use it as much as possible. But all objects from the back-end are in snake_case because that is how Django and Django Rest Framework does it by default, this is because Python uses it. It can be changed in the API, with something like [drf-camel-case](https://github.com/vbabiy/djangorestframework-camel-case), which just transforms it when requests are made.

Doing that with a current API with a lot of applications depending on it is not really viable. To do something like that, we would have to make a new API (v2), which would just be the same API with camelCase, which would be strange.

A common way of handling this is to remap the objects which are received from the back-end when they are fetched (and posted). But this just results in a lot of unnecessary extra code, _in my opinion_.

We are therefore left with somewhat of an abomination.
When the back-end is slowly replaced with other services, this will not be as much of an issue.